### PR TITLE
DOCS-7870 made edits

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/download-and-install-service-mesh.adoc [hunk 1]
```diff
@@ -15,21 +15,21 @@ To download the Anypoint Service Mesh client, type the following command at the
...
-// +
-// For example, type `cd /home/<your-user-name>/.bashrc` in a text editor.
-// . Add the following command to the last line of the file:
-// +
-// `export PATH="your-dir:$PATH"`, where *your-dir* is the directory you want to add.
-// . Save the `.bashrc` file.
-// . Restart your terminal.
+. Open the .bashrc file in your home directory.
++
+For example, type `cd /home/_your-user-name_/.bashrc)` in a text editor.
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/download-and-install-service-mesh.adoc:26]: If you like it, you can use this slightly simpler version:
```suggestion
For example, type `cd ~/.bashrc)` in a text editor.
```

Comment (wwarner-salesforce) [inline modules/ROOT/pages/download-and-install-service-mesh.adoc]: I think this is incorrect and should be changed as follows. If in doubt, check with the dev.
```suggestion
For example, type `nano ~/.bashrc` to open the file with the nano text editor.
```

File: modules/ROOT/pages/download-and-install-service-mesh.adoc [hunk 2]
```diff
@@ -15,21 +15,21 @@ To download the Anypoint Service Mesh client, type the following command at the
 
 `curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > asmctl && chmod +x asmctl`
 
+== Configure Your Environment Variable on Linux Systems
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/download-and-install-service-mesh.adoc]: I'd suggest this slight change, although the current version is OK. You might also check with the dev to see if this should be "Linux or macOS Systems".
```suggestion
== Configure Your PATH Variable on Linux Systems
```

File: modules/ROOT/pages/download-and-install-service-mesh.adoc (hunk 1 above)
Comment (dosten) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc]: 👍 I made the same correction in another PR but it was merged while I was commenting 😢

Comment (priyaayyar) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc]: Ok, will do!

File: modules/ROOT/pages/download-and-install-service-mesh.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc]: Will do!

Comment (dosten) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc]: **Unix-like**, or ***nix** IMO: https://en.wikipedia.org/wiki/Unix-like

Comment (priyaayyar) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc]: Tomas said Linux. Will check with him later

**Review by wwarner-salesforce — APPROVED** [review]

File: modules/ROOT/pages/download-and-install-service-mesh.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc]: Yeah, "Unix-like" or "*nix" would include Linux, macOS, and more rare systems such as BSD or the various old commercial mainframe flavors of Unix. "Unix-like" or "*nix" is the most technically correct option. On the other hand, I'm afraid some macOS and even Linux users might not understand that it applies to them. I'm pretty sure most people using a flavor of Unix more rare than Linux will know that "Linux or macOS" means them too. So, of the options, I like "Linux or macOS" the best, but I leave it up to Priya (and Tomas if she consults with him).

Comment (wwarner-salesforce) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc]: Oh-- I'm foolish. I just realized that Service Mesh probably only runs on Linux, in which case it should just say Linux.

File: modules/ROOT/pages/download-and-install-service-mesh.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/download-and-install-service-mesh.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/download-and-install-service-mesh.adoc]: Done! Thanks!

**Pull request merged**